test/inductor/test_compiled_optimizers.py

```diff
@@ -81,7 +164,10 @@ def test_fn(self):
             opt_eager.step()
 
         self.assertEqual(
-            list(model_eager.parameters()), list(model_compiled.parameters())
+            list(model_eager.parameters()),
+            list(model_compiled.parameters()),
+            atol=2e-5,
+            rtol=2e-3,
```

Comment:
these feel like pretty large tolerances...do you know why?

Reply:
Yeah, a single element of adadelta (1/100) was wrong when I didn't specify tolerance, I checked the inductor tests to see what other tolerances were common and landed on these. I can try tightening rtol though.

Reply:
I tightened rtol, and added a note on the specific test failure